### ISSUE TITLE
[BUG] When i open bot.py it closes immediately

### ISSUE THREAD

**Issue by eriel09** [Official-Husko/Husko-s-SteamWorkshop-Downloader#10]
Pls help

**Comment by Official-Husko** [comment]
Yes this is due to steamworkshop.io closing its backend system. it fails to do a backend check and therefor crashes. this will be solved in the next update when we integrated SteamCMD.

**Comment by eriel09** [comment]
Ok!

**Comment by adrylain** [comment]
Any general idea when the SteamCMD integration might be released?

**Comment by Official-Husko** [comment]
> Any general idea when the SteamCMD integration might be released?

well i planned to release it together with the rewrite directly which changes every bit of the system completely. It should be more userfriendly once its done and it will have more features but this also takes more time to code and we are working on a new backend system to serve locked mods from steam

**Comment by Official-Husko** [comment]
Just to mention it again this project is on hold as stated [here](https://github.com/Official-Husko/Husko-s-SteamWorkshop-Downloader#important) until a better method exists to download steamworkshop mods.
